core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -1389,6 +1389,12 @@ public void testJoinUsingDynamicTable() {
     sql(sql).expand(false).ok();
   }
 
+  @Test public void testSomeWithEquality() {
+    final String sql = "select empno from emp where deptno = some (\n"
+        + "  select deptno from dept)";
+    sql(sql).expand(false).ok();
+  }
+
```

Comment:
Would you please add a test for `=ANY`?

Reply:
@chunweilei ANY is alias for SOME, I am not sure if adding same test for ANY is worthwhile for this patch/request since it doesn't really touch that part of the code. BTW there is a test which uses =ANY is some.iq